modules/cdp-engine/src/lib.rs

```diff
@@ -1253,6 +1253,9 @@ impl<T: Config> Pallet<T> {
 
 		Ok(())
 	}
+	pub fn get_collateral_currency_ids() -> Vec<CurrencyId> {
+		CollateralParams::<T>::iter_keys().collect()
+	}
```

Comment:
This will break `on_initialize` benchmark. Not sure best practice for benchmarking unbounded iterators... is it even possible?